core/templates/dev/head/domain/statistics/PlaythroughImprovementCardObjectFactory.js

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Factory for creating Playthrough Cards in the Improvements Tab.
+ */
+
+oppia.factory('PlaythroughImprovementCardObjectFactory', [
+  'HtmlEscaperService', 'ImprovementActionButtonObjectFactory',
+  'PlaythroughIssuesService',
+  function(
+      HtmlEscaperService, ImprovementActionButtonObjectFactory,
+      PlaythroughIssuesService) {
+    /** @constructor */
+    var PlaythroughImprovementCard = function(playthroughIssue) {
+      var that = this;
+      var archiveThis = function() {
```

Comment:
Does "archive" mean "deleted"? Becuase the two are kinda different things ("archive" suggests "still retrievable").

Reply:
Done — it is now "Discard".